cookbooks/aws-parallelcluster-slurm/libraries/helpers.rb

```diff
@@ -114,3 +114,29 @@ def setup_munge_compute_node
 
   enable_munge_service
 end
+
+def package_version(pkg_glob)
+  version = 'unknown'
+  cmdline = "rpm -qa --queryformat '%{NAME}" "%{VERSION}\n' #{pkg_glob}"
+  if platform?('ubuntu')
+    cmdline = "dpkg-query -W  -f='${Package} ${Version}\n' #{pkg_glob}"
```

Comment:
We may consider using `${Source:Upstream-Version}` instead of `${Version}` to remove the `*ubuntu*` suffix in `${Version}`.

Reply:
Also, why do we need the package name in the output of `cmdline`? I don't see it used elsewhere.

Reply:
After discussing pros and cons we decided to fix the version of the lua library